src/network/networkd-link.c

```diff
@@ -199,6 +199,21 @@ static bool link_proxy_arp_enabled(Link *link) {
         return true;
 }
 
+static bool link_ipv6_proxy_ndp_enabled(Link *link) {
```

Comment:
"enabled" means "configured" or "set" here, right? It can be both enabled and disabled, if the function returns true. The name is misleading.